src/pages/OnBoarding/util/OnBoardingContext.jsx

```diff
@@ -114,12 +114,31 @@ export const OnBoardingProvider = ({ children, initStep, onFinish }) => {
   // console.log({ ynputConnect })
   // step 2
   const [userForm, setUserForm] = useState(initUserForm)
-  // step 3
+  // step 5
```

Comment:
This counting looks odd - just no variables/states for step 3 and 4?

Reply:
no states for them :)